RocketChat/Rocket.Chat
# fix: Add support for non-alphanumeric letters in LDAP Sync

## Proposed changes (including videos or screenshots)





## Issue(s)

Close #24972
## Steps to test or reproduce

- Create a new user in LDAP with a username containing special letters in it: e.g -> ä
- Sync in Rocket.Chat
- It will sync successfully, but when you check the created user he doesn't have a username

## Further comments

[TC-373]


[TC-373]: https://rocketchat.atlassian.net/browse/TC-373?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
